api/base/serializers.py

```diff
@@ -143,92 +173,141 @@ def get_attribute(self, obj):
         return field_source_method(auth=auth)
 
 
-class JSONAPIHyperlinkedIdentityField(ser.HyperlinkedIdentityField):
+class RelationshipField(ser.HyperlinkedIdentityField):
```

Comment:
OK, I see now that subclassing `HyperlinkedIdentityField` allows us to re-use some the `__init__` and `to_representation` code, so I think this is fine.